x-pack/package.json

```diff
@@ -130,6 +130,7 @@
     "ts-loader": "^5.2.2",
     "typescript": "^3.0.3",
     "vinyl-fs": "^3.0.2",
+    "why-did-you-update": "^1.0.6",
```

Comment:
Do we need this in the package? You're not using it anywhere, and it seems easy enough to only pull it in locally when you want to diagnose re-renders.